tokio-test/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "tokio-test"
+
+# When releasing to crates.io:
+# - Update html_root_url.
+# - Update doc url
+#   - Cargo.toml
+#   - README.md
+# - Update CHANGELOG.md.
+# - Create "v0.1.x" git tag.
+version = "0.1.0"
+authors = ["Carl Lerche <me@carllerche.com>"]
+license = "MIT"
+repository = "https://github.com/tokio-rs/tokio"
+homepage = "https://tokio.rs"
+documentation = "https://docs.rs/tokio-test/0.1.0/tokio_test"
+description = """
+Tokio and futures based testing utilites
+"""
+categories = ["asynchronous", "testing"]
+
```

Comment:
I think we should probably add a `publish = false` here until we're ready to release an 0.1, to avoid doing so accidentally?